test/test_utils.py

```diff
@@ -121,3 +125,34 @@ def test_get_string_to_sign_no_optional_params():
     )
 
     assert result == ("GET\n" "\n" "\n" "1388534400\n" "/bucket/objectname")
+
+
+def test_aws_get_presigned_url():
+    """
+    Test that we can get a presigned url from a bucket
+    """
+
+    s3 = boto3.client("s3", aws_access_key_id="", aws_secret_access_key="")
```

Comment:
I'm not seeing where this is mocked, so I'm actually not sure how this is passing

Reply:
I'm not sure it need mocks 🤔 the `generate_presigned_url()` in boto3 is a local operation